bundles/af-core/src/test/resources/form/checkboxgroup/exporter-checkboxgroup-datalayer.json

```diff
@@ -9,8 +9,14 @@
   "readOnly": false,
   "enforceEnum": true,
   "enumNames": [
-    "Cat Person",
-    "Dog Person"
+    {
+      "richText": false,
```

Comment:
Please don't add this value unless explicitly specified in author

Reply:
done